src/scripts/safari/SafariPokemonList.ts

```diff
@@ -40,6 +42,14 @@ class SafariPokemonList {
             new SafariEncounter('Cubone', 10),
             new SafariEncounter('Marowak', 5),
             new SafariEncounter('Tangela', 4),
+            // Water
```

Comment:
Magikarp is missing from the encounter list.

But, if we're going slightly off canon, should the dragons be in here? It would make the dragon egg obsolete.

Reply:
I'll add Magikarp.

As for Dratini and Dragonair you still need to catch them before they can appear (same condition as Friend Safari special Pokémon) so I don't think it would make the egg obsolete.
I can remove them if you prefer, however.

Reply:
Serves me right for commenting before fully understanding.

Adding Magikarp and keeping the dragons locked behind already being obtained should be totally fine, thanks!